packages/playground/data-liberation/tests/import/blueprint-import.json

```diff
@@ -1,15 +1,14 @@
 {
 	"$schema": "../../../blueprints/public/blueprint-schema.json",
-	"constants": {
-		"WP_DEBUG": true,
-		"WP_DEBUG_DISPLAY": true,
-		"WP_DEBUG_LOG": true
-	},
 	"login": true,
 	"steps": [
 		{
 			"step": "activatePlugin",
 			"pluginPath": "data-liberation/plugin.php"
+		},
+		{
+			"step": "runPHP",
+			"code": "<?php require_once 'wordpress/wp-load.php'; $base = '/wordpress/wp-content/plugins/data-liberation/';\nrequire $base . 'vendor/autoload.php';\ntry {\n$arguments = [\n'--stderr',\n'--configuration', $base . 'phpunit.xml'\n];\n$res = (new PHPUnit\\TextUI\\Application())->run($arguments);\nif ( $res !== 0 ) {\ntrigger_error('PHPUnit failed', E_USER_ERROR);\n}\n} catch (Throwable $e) {\ntrigger_error('PHPUnit failed: ' . $e->getMessage(), E_USER_ERROR);\n};"
```

Comment:
Cool idea! This will suffice for starters, but here's something if you'd like to the next level. What would it take to go from this to something more like a typical CLI command, e.g. `cli --blueprint=... --mount=... run vendor/bin/phpunit --configuration phpunit.xml --stderr`?

Reply:
Great idea. Didn't touched the CLI in this first phase.